web/app/css/line-graph.css

```diff
@@ -21,3 +21,7 @@
 .line-p99 {
   stroke: var(--latency-p99);
 }
+
+.flash {
+  fill: var(--latency-p50);
```

Comment:
I think ideally we would name this blue something descriptive, and not have "--latency-p50" be the name of a color, though I realize this is a problem throughout our blues...

Reply:
I can make a follow-up issue with renaming all the colors across the project, should I?

Reply:
that would be great!

Reply:
done!